### PR TITLE
BD-2733: Building out the compliance doc portal

### PR DESCRIPTION
**Pull request by josh-mccrowell-braze**

### CODE REVIEW

File: _docs/_hidden/compliance/home.md [hunk 1]
```diff
@@ -1,56 +1,36 @@
...
 
-guide_menu_title: "Legal"
+  ## What is included in the Compliance Documentation?
+
+  The Compliance Documentation below sets forth specific terms applicable to your purchased product, channel, feature, functionality or service:
+
+  * For Braze Services functionality that allows customers to interact with, integrate with or access the product, website, application or service of a Third-Party Provider, the Compliance Documentation contains the Third-Party Provider terms applicable to your use of such functionality; and
+
+  * Any general industry practices and standards that Braze customers are required to comply with for the use of such Braze product, channel, feature, functionality or service.
+guide_top_text2: "<h2>Updates to the Compliance Documentation</h2><br>You can subscribe to receive updates to our documentation (including the Compliance Documentation) through Braze’s GitHub repository."
```
Comment (ats-91) [inline _docs/_hidden/compliance/home.md:19]: We would like to add an hyperlink to the expression "Braze’s GitHub repository" - https://github.com/braze-inc/braze-docs (we believe it is the right link where the updates to the Documentation are inserted/customers can create an account and subscribe to notifications. but let us know if this is not the right link).

File: _docs/_hidden/compliance/home.md [hunk 2]
```diff
@@ -1,56 +1,36 @@
...
-  fa_icon: fa-solid fa-lock
-- name: Braze Security Certifications
-  link: /docs/developer_guide/disclosures/security_qualifications/#soc-2-examination
-  fa_icon: fa-solid fa-lock
 
 ---
-
+<br><br>
 {% alert important %}
 **Legal Disclaimer**<br>
```
[the remark this answers is not in the document]
Comment (josh-mccrowell-braze) [reply _docs/_hidden/compliance/home.md:35]: Done

File: _docs/_hidden/compliance/home.md (hunk 1 above)
Comment (josh-mccrowell-braze) [reply _docs/_hidden/compliance/home.md:19]: Correct, you can "Watch" this repository and be updated whenever there's a new deployment (twice weekly)

**Comment by josh-mccrowell-braze** [comment]
@lydia-xie Could you please give this a review before the deployment today? I know that these pages aren't in line with our style guide, but they are precisely what Legal wants.

**Review by lydia-xie — APPROVED** [review]
looks good to me!

**Pull request merged**